pubspec.yaml

```diff
@@ -1,15 +1,19 @@
 name: csslib
-version: 0.16.3-dev
+version: 0.17.0-nullsafety-dev
 
 description: A library for parsing CSS.
 homepage: https://github.com/dart-lang/csslib
 
 environment:
-  sdk: '>=2.2.0 <3.0.0'
+  sdk: '>=2.10.0-2.0.dev <2.10.0'
 
 dependencies:
-  source_span: ^1.4.0
+  source_span: ^1.8.0-nullsafety
 
 dev_dependencies:
   pedantic: ^1.0.0
```

Comment:
there is a null safe pedantic as well, might as well require it even if we don't use `unawaited`?

Reply:
Shouldn't hurt anything either way. I can update it.